doc/sphinx/proofs/writing-proofs/proof-mode.rst

```diff
@@ -81,7 +183,6 @@ list of assertion commands is given in :ref:`Assertions`. The command
       even incur a memory overflow.
 
 .. cmd:: Save @ident
-   :name: Save
 
    Saves a completed proof with the name :token:`ident`, which
```

Comment:
We should either indent `Save` to make it clear that it is a variant of `Qed` or be clearer in the description that it does the same thing as `Qed` except for the naming. E.g.:
```suggestion
   Similar to :cmd:`Qed`, except that the proof term is added to the global context with the name :token:`ident`, which
```